tests/Unit/Options/Test_Options.cpp

```diff
@@ -767,9 +785,12 @@ void test_options_format() {
   const auto check = [](auto opt) noexcept {
     using Opt = decltype(opt);
     Options<tmpl::list<Opt>> opts("");
-    INFO(opts.help());
+    INFO("Help string:\n"
+         << opts.help() << "\n\nExpected to find:\n"
+         << "  "s + Opt::expected + "\n");
     // Add whitespace to check that we've got the entire type
-    CHECK(opts.help().find("  "s + Opt::expected + "\n") != std::string::npos);
+    CHECK(opts.help().find(std::string(Opt::expected) + "\n") !=
```

Comment:
Should this line be `CHECK(opts.help().find("type=" + std::string(Opt::expected) + "\n") !=`  since now the opts.help() function prints  'type='?  And same with the info string above. Please pardon me if I'm not understanding something because I'm certainly not an expert on the options code.

Reply:
I will try this, I agree that it should work :)